<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayImportDlg</class>
 <widget class="QDialog" name="DayImportDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>每日数据导入</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="title">
      <string>数据导入</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,2,2,1">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>客户成交信息表路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_cj"/>
          </item>
          <item>
           <widget class="QToolButton" name="tb_cj">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pb_cj">
            <property name="text">
             <string>开始导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="pro_cj">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>客户持仓信息表路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_cc"/>
          </item>
          <item>
           <widget class="QToolButton" name="tb_cc">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pb_cc">
            <property name="text">
             <string>开始导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="pro_cc">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>客户资金风险表路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_zj"/>
          </item>
          <item>
           <widget class="QToolButton" name="tb_zj">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pb_zj">
            <property name="text">
             <string>开始导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="pro_zj">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_cancel">
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
